types/ol/proj/index.d.ts

```diff
@@ -0,0 +1,3 @@
+import * as ol from 'openlayers';
+
+export default ol.proj;
```

Comment:
I believe this should be an `export = ol.proj`. [Please read more here from the DefinitelyTyped REAMDE](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/fbba9f55beab742b9d1c0fa9ef714e0771ad9d05#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default)

Additionally, to keep your test file working the same, I assume `esModuleInterop` must have been set.

Reply:
I added this file by referring to the other d.ts files. The other files in this library all uses the `export default` format. Should I make this change?